### PR TITLE
Use optional field for observation value

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by Spaceenter — APPROVED** [review]

File: internal/proto/place.pb.go [hunk 1]
```diff
@@ -1396,9 +1396,9 @@ type RelatedPlacesInfo_Ranking_RankInfo struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	Rank      int32   `protobuf:"varint,1,opt,name=rank,proto3" json:"rank,omitempty"`
-	Value     float64 `protobuf:"fixed64,2,opt,name=value,proto3" json:"value,omitempty"`
-	PlaceDcid string  `protobuf:"bytes,3,opt,name=place_dcid,json=placeDcid,proto3" json:"place_dcid,omitempty"`
+	Rank      int32    `protobuf:"varint,1,opt,name=rank,proto3" json:"rank,omitempty"`
+	Value     *float64 `protobuf:"fixed64,2,opt,name=value,proto3,oneof" json:"value,omitempty"`
```
Comment (pradh) [inline internal/proto/place.pb.go:1400]: has pointer and a `oneof` flag :)

**Review by pradh — APPROVED** [review]

File: internal/server/stat/golden/get_stat_set_within_place/epa_facility.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
         },
         "epaGhgrpFacilityId/1000033": {
           "date": "2019",
+          "value": 0,
```
Comment (pradh) [inline internal/server/stat/golden/get_stat_set_within_place/epa_facility.json:12]: Its believable that emissions (Annual_Emissions_GreenhouseGas_NonBiogenic) could be 0?

Has a downward trend: https://screenshot.googleplex.com/6Jq55JRgMKgknXq

Comment (shifucun) [reply internal/server/stat/golden/get_stat_set_within_place/epa_facility.json:12]: I checked this and other diffs, all looking good!

**Comment by shifucun** [comment]
Upgraded Golang to 1.19 to use optional in proto3. Also updated linter version and cleaned up lint errors.

**Pull request merged**